HOMEWORK_8/EX4.CPP

```diff
@@ -0,0 +1,33 @@
+#include <iostream>
+
+const float PI = 3.14;
```

Comment:
prefer using predefined pi value, for example M_PI from cmath

Reply:
done